pubcloud cosmetic fixes (#2662)
uyuni-project/uyuni-docs: modules/specialized-guides/pages/public-cloud-guide/byos/byos-overview.adoc

== Bring your own subscription (BYOS)

> Bring your own subscription (BYOS) images are useful if you already have a support contract with SUSE and want to move your workloads to the public cloud.

Reviewer: ```suggestion
{byoslongform} ({byos}) images are useful if you already have a support contract with SUSE and want to move your workloads to the public cloud.
```

> An instance launched from a BYOS image is equivalent to a physical machine that just received a SUSE Linux Enterprise Server installation from the {sles} installation image.

Reviewer: ```suggestion
An instance launched from a {byos} image is equivalent to a physical machine that just received a SUSE Linux Enterprise Server installation from the {sles} installation image.
```

After creation:

* use ``registercloudguest`` to register the instance with the SUSE-operated update infrastructure in the cloud framework, or
* use the command ``SUSEConnect`` to register the system with {scc} using the entitlements you already have.

Connect your system to your own running RMT or {productname} infrastructure in the same way you connect systems in your data center.

> BYOS instances make it easier to manage extensions such as ``LTSS`` or ``kernel live-patching``.

Reviewer: ```suggestion
{byos} instances make it easier to manage extensions such as ``LTSS`` or ``kernel live-patching``.
```
